src/libraries/System.Private.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Function.cs

```diff
@@ -44,7 +44,7 @@ internal Function(IntPtr jsHandle, bool ownsHandle) : base(jsHandle, ownsHandle)
         /// <returns>The result of calling the function with the specified `this` value and arguments.</returns>
         /// <param name="thisArg">Optional (null value). The value of this provided for the call to a function. Note that this may not be the actual value seen by the method: if the method is a function in non-strict mode, null and undefined will be replaced with the global object and primitive values will be converted to objects.</param>
         /// <param name="argsArray">Optional. Arguments for the function.</param>
-        public object Call(object? thisArg = null, params object[] argsArray)
+        public object Call(JSObject? thisArg = null, params object[] argsArray)
```

Comment:
fwiw, there are cases where you might want to invoke a method on something that isn't already a jsobject - for example you could pass in a C# string to call ```toLowerCase``` on it. I'm fine with deciding not to support it but there's an asymmetry introduced by not making ```this``` able to accept any type since it can be any type in JS.

Reply:
That first parameter being `thisAgrg` hits me 90% time when I use it, because I forget to pass `null` here and then get stupid behavior a I have args shifted. This helps me a little bit, but I'm ok to revert it or to do something else, if you have an idea.

Reply:
maybe make thisArg not optional?